test/plugin/test_out_google_cloud.rb

```diff
@@ -339,10 +339,125 @@ def test_statusz_endpoint
     WebMock.disable_net_connect!(allow_localhost: true)
     # TODO(davidbtucker): Consider searching for an unused port
     # instead of hardcoding a constant here.
-    d = create_driver('statusz_port 5678')
+    d = create_driver(%(
+      statusz_port                  5678
```

Comment:
Shouldn't this config live in `Constants`?

Reply:
Done.